web-app/css/elurikkus-common.css

```diff
@@ -324,7 +315,7 @@ input:focus, textarea:focus, select:focus {
 /* 1920px / 2 - vertical UI space. */
 /* 2560px / 2 - vertical UI space. */
 body {
-    margin: 12px;
+    margin: 30px;
```

Comment:
That's how we roll now. Also, should remove bootstrap container from everything - we get enough spacing from this and anything elses messes with menu alignment